velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -52,5 +53,11 @@ int main(int argc, char** argv) {
       "replace",
       "might_contain",
       "unix_timestamp"};
-  return FuzzerRunner::run(FLAGS_seed, skipFunctions);
+
+  // This configuration is necessary for evaluating the 'spark_partition_id'
```

Comment:
"This configuration is necessary for evaluating" is a bit too long. Consider shortening to "Required by spark_partition_id function."